code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -83,6 +83,30 @@
 		if("cry")
 			message = "<B>[comm_paygrade][src]</B> cries."
 
+		if("dab")
+			message = "<B>[comm_paygrade][src]</B> dabs."
+			log_message(message, LOG_EMOTE)
+
+			for(var/mob/M in dead_mob_list)
```

Comment:
It shouldn't make much of a difference as this is processed in a blink. But if you are meticolous about it you should add an emote_backfire(act) proc after the message is sent, because this is kind of bad.